gms/gossiper.cc

```diff
@@ -855,6 +855,45 @@ future<> gossiper::failure_detector_loop() {
     logger.info("failure_detector_loop: Finished main loop");
 }
 
+void gossiper::duplicate_live_endpoints_on_change() {
```

Comment:
s/duplicate/replicate (standard terminology used elsewhere)

Reply:
I'd also add `assert(this_shard_id() == 0)`

Reply:
Will do, but I used "duplicate" as it was like that in the comments/commits.